tempus_fugit_minecraft/main.py

```diff
@@ -430,6 +438,33 @@ def process_entire_queue(self):
         """
         while self.queue:
             self._dequeue()
+    
+    
+    def generate_clouds(self,n,num_of_clouds = 250):
+        """
+        Generate the position of the clouds on the sky.
+        
+        Inputs: n = 1/2 size of the world.
+                num_of_clouds = default clouds to be generated =250
+        
+        Output: return a list of lists; each inner list represents a set of cloud blocks.
+        """
+        o = n - 10
+        clouds = list()
+        for _ in xrange(num_of_clouds):
+            cloud_center_x = random.randint(-o, o)  # x position of the cloud
+            cloud_center_z = random.randint(-o, o)  # z position of the cloud
+            cloud_center_y = 20                     # y position of the cloud (height)
```

Comment:
In the next update, I suggest giving random y values. For example, y=18 to 22 to make the clouds generate at different heights. Look at issue #44.